legal-api/report-templates/template-parts/common/v2/styleMail.html

```diff
@@ -349,6 +349,14 @@
 		text-align: start;
 	}
 
+	.retrieved-date-time {
+		font-family: 'BCSans-Italic', sans-serif !important;
+		font-size: 13px;
+		line-height: 15px;
+		color: #212529;
+		padding: 1rem 1rem 0.6rem 1rem;
+	}
```

Comment:
Do you have an example where this is used?

Reply:
Can you work backwards from "which templates uses this style file" to "which code uses the templates" to "which features use that code"?